lib/cartopy/crs.py

```diff
@@ -146,6 +146,13 @@ def domain(self):
             domain = self._domain = sgeom.Polygon(self.boundary)
         return domain
 
+    def _repr_html_(self):
+        import matplotlib.pyplot as plt
```

Comment:
Needs a comment to explain context.
? something like ? "enables self-display in a Jupyter notebook"